code/game/machinery/computer/cloning.dm

```diff
@@ -358,35 +358,31 @@
 		return
 	if(scan_brain && !can_brainscan())
 		return
-	if((isnull(subject)) || (!(ishuman(subject))) || (!subject.dna) || (NO_SCAN in subject.dna.species.species_traits))
```

Comment:
This message at the end, NO_SCAN in subject dna, it gets applied to people when they are absorbed by a changeling, and prevents cloning from happening if the cloner is not upgraded enough, could use a message why the person failed to be scanned correctly

Reply:
Isn't that what this condition is?

```
if((NOCLONE in subject.mutations) && src.scanner.scan_level < 2)
```

That's still there.  The condition I removed could actually never be hit as far as I could tell.
```
if(scan_brain && !subject.get_int_organ(/obj/item/organ/internal/brain))
```

That's the condition I removed.  There's a condition which checks to see if they have no brain removed a few checks above so that one would always be hit before this one was ever checked.

Reply:
ah my mistake, I am blind, saw the red text under it and presumed it was removed as well